src/Kunstmaan/TranslatorBundle/Service/Command/Importer/ImportCommandHandler.php

```diff
@@ -37,11 +37,22 @@ class ImportCommandHandler extends AbstractCommandHandler
      */
     public function executeImportCommand(ImportCommand $importCommand)
     {
-        if ($importCommand->getGlobals() || $importCommand->getDefaultBundle() === false || $importCommand->getDefaultBundle() === null) {
-            return $this->importGlobalTranslationFiles($importCommand);
+        $amount = 0;
+        $defaultBundleNotSet = $importCommand->getDefaultBundle() === false || $importCommand->getDefaultBundle() === null;
+
+        if ($importCommand->getGlobals()) {
+            if ($defaultBundleNotSet) {
+                return $this->importGlobalTranslationFiles($importCommand);
+            }
+
+            $amount = $this->importGlobalTranslationFiles($importCommand);
+        } elseif ($defaultBundleNotSet) {
+            $importCommand->setDefaultBundle('all');
         }
 
-        return $this->importBundleTranslationFiles($importCommand);
+        $amount += $this->importBundleTranslationFiles($importCommand);
+
+        return $amount;
```

Comment:
Maybe the code can be restructed a bit to avoid the double `importGlobalTranslationFiles` function call and the elseif. 

```php
        if ($importCommand->getGlobals()) {
            $amount = $this->importGlobalTranslationFiles($importCommand);
            if ($defaultBundleNotSet) {
                return $amount;
            }
        } 
        
        if ($defaultBundleNotSet) {
            $importCommand->setDefaultBundle('all');
        }

        $amount += $this->importBundleTranslationFiles($importCommand);

        return $amount;
```